account_avatax/models/avalara_salestax.py

```diff
@@ -136,6 +136,10 @@ def _get_avatax_supported_countries(self):
         help="Allows ean13 to be reported in place of Item Reference"
         " as upc identifier.",
     )
+    avatax_company_id = fields.Char(
+        'Company ID',
+        help="The company ID as defined in the Admin Console of AvaTax",
+    )
```

Comment:
This info is already there, as `company_code `

Reply:
avatax_company_id is their reference, used in api calls and useful to have for the ecom integrations. company_code is our reference value

![image](https://user-images.githubusercontent.com/6782997/100658187-a8dba580-331c-11eb-82db-238320efad62.png)